src/edu/stanford/nlp/sempre/InfiniteListValue.java

```diff
@@ -0,0 +1,66 @@
+package edu.stanford.nlp.sempre;
+
+import java.util.*;
+
+import fig.basic.LispTree;
+
+/**
+ * Represent a list of infinitely many values.
+ *
+ * The list is represented by a List of Objects.
+ *
+ * @author ppasupat
+ */
+public class InfiniteListValue extends Value {
```

Comment:
Yeah... this is a hack to represent the denotations of the form (< 5), (!= fb:cell.greece), and (and (> 10) (<= 20)). Rename?